src/main/java/ro/vladutit/Don/t/forget/v2/controller/ProfileController.java

```diff
@@ -0,0 +1,24 @@
+package ro.vladutit.Don.t.forget.v2.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import ro.vladutit.Don.t.forget.v2.model.CustomUserDetails;
+import ro.vladutit.Don.t.forget.v2.service.UserService;
+
+@Controller
+public class ProfileController {
+
+    @Autowired
+    public UserService userService;
+
+    @RequestMapping("/profile")
```

Comment:
/user/:id

Reply:
Because I didn't have public profiles, just private profiles, so we can view only our information profile user. In this case, and also the _id_ or _username_ I can now with an annotation from _Spring_ (`@AuthenticationPrincipal`), was ok, in my opinion, without specifying the _id_ or the _username_ in the _route_.